libyul/ObjectParser.h

```diff
@@ -67,6 +69,7 @@ class ObjectParser: public langutil::ParserBase
 	void addNamedSubObject(Object& _container, YulString _name, std::shared_ptr<ObjectNode> _subObject);
 
 	Dialect const& m_dialect;
+	Parser::DebugAttributeCache::Ptr m_debugAttributeCache;
```

Comment:
Do these optimizer changes related to `DebugAttributeCache` really belong in a PR that's adding a new test type? Shouldn't they be a part of some other PR specific to debug info?